app/helpers/bids_helper.rb

```diff
@@ -6,4 +6,10 @@ def content_for_row(row_number, bid)
       number_to_currency(bid.amount)
     end
   end
+
+  def user_is_winning_bidder?(user, auction)
+    return false if !auction.current_bid?
+
+    user.id == auction.current_bid.bidder_id
+  end
```

Comment:
I would say you want to stick with presenters or switch to helpers. They perform the same task in two different ways. 

* Helpers are mixins and therefore end up as bald methods in the views. They can get hard to extract, when refactoring. Because they are methods in an unknown object, their dependencies can sometimes be hard to suss out. Also their are not easily available to non-view places like models and APIs (largely due to the unknown dependencies).
* Decorators are tightly scoped to a model, AR or Poro. This is what I had going on. They are limited that way and start to feel awkward as the view expands beyond the strict CRUD land.
* When that happens another option, and this is more a naming and conceptual than anything else, is a view model. View models are responsible for presenting the needed data to your view, and are tied to the concept of the view needs rather than the model needs.

Anyway, I would go with POROs for view logic or helpers but not both.